tests/rule-engine/ForwardChainerUTest.cxxtest

```diff
@@ -38,21 +38,20 @@ public:
 		// Disable the AF mechanism during testing!
 		_as.set_attentional_focus_boundary(AttentionValue::MINSTI);
 
-		eval.eval("(add-to-load-path \"..\")");
```

Comment:
Don't remove this: its needed to run the stand-alone unit test for the command line.  For example, if you `cd build` to your build directory, and then say `./tests/rule-engine/ForwardChainerUTest`  then the test will pass with the above line, and it will fail without the above line.